plugins/node/opentelemetry-instrumentation-fastify/src/instrumentation.ts

```diff
@@ -27,10 +27,12 @@ import {
   safeExecuteInTheMiddle,
 } from '@opentelemetry/instrumentation';
 import { SemanticAttributes } from '@opentelemetry/semantic-conventions';
-import type { HookHandlerDoneFunction } from 'fastify/types/hooks';
-import type { FastifyInstance } from 'fastify/types/instance';
-import type { FastifyReply } from 'fastify/types/reply';
-import type { FastifyRequest } from 'fastify/types/request';
+import type {
+  HookHandlerDoneFunction,
+  FastifyInstance,
+  FastifyRequest,
+  FastifyReply,
+} from 'fastify';
```

Comment:
are the changes here intended? :thinking:

Reply:
Yes, while writing this guide I scanned the existing code base and found those imports (which are related to the changes in the guidelines) to be not ideal, and fixed that on the way.